examples/test_rocm/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.1.3)
 
-project (test_rocm)
```

Comment:
Im confused.  Can you help me understand the changes to the example CMakeLists?

Reply:
That project command isnt needed and causes an infinite cache reload when using the Trilinos compiler settings on the parent level cmakelist

Reply:
Ahh.  Good to know.  infinite cache reload is bad lol.